policyengine_us/variables/gov/states/az/tax/income/exemptions/az_parents_grandparents_exmeption.py

```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class az_parents_grandparents_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona parents and grandparents exemption"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.az.tax.income.exemptions
+        person = tax_unit.members
+        # The exemption is provided for parents and grandparents who receive over a percentage of their care and support
+        # payments from the filer.
+        care_and_support_payment = person("care_and_support_payment", period)
+        care_and_support_costs = person("care_and_support_costs", period)
+        support_payment_ratio = np.zeros_like(care_and_support_costs)
+        mask = care_and_support_costs != 0
+        support_payment_ratio[mask] = (
+            care_and_support_payment[mask] / care_and_support_costs[mask]
+        )
+        payment_eligible = (
+            support_payment_ratio > p.parent_grandparent.cost_rate
+        )
+        # Eligible parents of ancestors of parents have to be at or over a certain age threshold
+        # as well as cohabiting with the filer
+        age = person("age", period)
+        age_eligible = age >= p.parent_grandparent.min_age
+
+        eligible_parent = age_eligible & payment_eligible
+
+        eligible_grandparent = age_eligible & payment_eligible
```

Comment:
should `eligible_parent` and `eligible_grandparent` have the same formula?

Reply:
I think the criteria to justify the eligibility of parents and grandparents are the same. 
![image](https://github.com/PolicyEngine/policyengine-us/assets/112495414/097e8537-e93b-42dd-a482-87e3c00522b7)